src/lighteval/main_accelerate.py

```diff
@@ -48,9 +48,8 @@ def main(args):
     env_config = EnvConfig(token=TOKEN, cache_dir=args.cache_dir)
     evaluation_tracker = EvaluationTracker(
         output_dir=args.output_dir,
-        hub_results_org=args.results_org,
```

Comment:
Why did you remove this one ? There is a check at init that fails if this is not set while push_to_hub is set

Reply:
Mhh I added it back a minute ago, I could'nt see it before